src/java/org/apache/cassandra/cql3/statements/schema/AlterTypeStatement.java

```diff
@@ -205,6 +223,9 @@ UserType apply(KeyspaceMetadata keyspace, UserType userType)
         }
 
         private final UTName name;
+        private final boolean ifExists;
```

Comment:
I would rename all these `ifExists` to `ifTableExists` for readability throughout the PR.

Reply:
hmmm, I had that earlier, but it seems that the consensus from an earlier PR ( that I made on top of 4.0 branch unfortunately) was to convert them to ifExists. The understanding was that the meaning of the variable can be infered from context. Here are the previous review comments.

https://github.com/apache/cassandra/pull/1319#discussion_r754352408
https://github.com/apache/cassandra/pull/1319#discussion_r755154353
https://github.com/apache/cassandra/pull/1319#discussion_r755155148
https://github.com/apache/cassandra/pull/1319#discussion_r754354472
https://github.com/apache/cassandra/pull/1319#discussion_r754355124
https://github.com/apache/cassandra/pull/1319#discussion_r754353396
https://github.com/apache/cassandra/pull/1319#discussion_r754352775

Reply:
Oh Benjamin and myself always disagree on the small readability nits lol!